app/src/main/java/org/schabi/newpipe/util/NavigationHelper.java

```diff
@@ -155,57 +158,29 @@ public static void playOnBackgroundPlayer(final Context context,
         intent.putExtra(Player.PLAYER_TYPE, MainPlayer.PlayerType.AUDIO.ordinal());
         ContextCompat.startForegroundService(context, intent);
     }
-
-    public static void enqueueOnVideoPlayer(final Context context, final PlayQueue queue,
-                                            final boolean resumePlayback) {
-        enqueueOnVideoPlayer(context, queue, false, resumePlayback);
-    }
-
-    public static void enqueueOnVideoPlayer(final Context context, final PlayQueue queue,
-                                            final boolean selectOnAppend,
-                                            final boolean resumePlayback) {
-
+    /* ENQUEUE */
+    public static void enqueueOnPlayer(final Context context, final PlayQueue queue) {
```

Comment:
As discussed above, we apparently still need to be able to enqueue and switch to a different player at the same time. Therefore I would just have this function take `playerType` as a parameter...
```suggestion
    public static void enqueueOnPlayer(final Context context, final PlayQueue queue, final PlayerType playerType) {
```

Reply:
8cbfcc4fa02f4668c427d48d42f7978f5b9a2a80